web/js/components/charting/chart-component.js

```diff
@@ -144,9 +144,10 @@ function ChartComponent (props) {
           </div>
         </div>
         <div className="charting-discalimer">
```

Comment:
I realize this isn't your error but can we fix this typo?

discalimer --> disclaimer